README.md

```diff
@@ -1,15 +1,35 @@
-# Arcaflow Engine
+# Arcaflow: The Noble Workflow Engine
+<img align="left" width="200px" style="padding-right: 2em; padding-bottom: 2em" alt="Arcaflow logo showing
+a waterfall and a river with 3 trees symbolizing the various plugins"
+src="https://github.com/arcalot/.github/raw/main/branding/arcaflow.png">
 
-The Arcaflow Engine allows you to run workflows using container engines, such as Docker,
-Podman, or Kubernetes, or for supported plugins using Python directly. The Arcaflow SDK
-is available for [Python](https://arcalot.io/arcaflow/creating-plugins/python/) and
+**Arcaflow** is a highly-flexible and portable **workflow system** that helps
+you to build simple or complex pipelines of actions with data passing between the
+actions. The **data is validated** according to schemas along the way to make sure
+there is no corrupt data, catching potential failure conditions before they happen.
+Arcaflow runs on your laptop, a jump host, or in a CI system requiring only the Arcaflow
+engine binary, a workflow definition in YAML, and a compatible container runtime.
+
+[Complete Arcaflow Documentation](https://arcalot.io/arcaflow)
+
+![image](arcaflow-basic-demo.gif)
+
+# The Arcaflow Engine
+
+The Arcaflow Engine is the core execution component for workflows. It allows you to run
```

Comment:
This feels awkward, like the point of a workflow is to run docker or podman. Rather, the point of the workflow is to run containerized plugins; and we support podman, docker, and k8s to launch them.

"It allows you to use services provided by containerized Arcaflow 'plugins'. The Arcaflow engine can be configured to run plugins using Docker, Podman, or Kubernetes."